contracts/standard/rng/BlockhashRNGFallbackRange.sol

```diff
@@ -0,0 +1,48 @@
+ /**
+ *  @authors: [@clesaege]
+ *  @reviewers: [@remedcu]
+ *  @auditors: []
+ *  @bounties: []
+ *  @deployments: []
+ */
+
+pragma solidity ^0.4.15;
+
+import "./BlockhashRNG.sol";
+
+/**
+ *  @title Random Number Generator using blockhash with fallback from the last 256 blockhash mined.
+ *  @author Clément Lesaege - <clement@lesaege.com>
+ *
+ *  Random Number Generator returning the blockhash with a backup behaviour.
+ *  This contract implements the RNG standard and gives parties incentives to save the blockhash to avoid it to become unreachable after 256 blocks.
+ *  In case no one called it within the 256 blocks, it returns the blockhash from any one of the last mined 256 blocks.
+ *  Thus allowing the contract to still access the blockhash even after 256 blocks.
```

Comment:
"access the blockhash" -> "return a value"

Reply:
Changed